### PR TITLE
Correct Quarkus version & examples modules

### PR DESCRIPTION
**Pull request by radtriste**
Looks like crossmerge of PRs is problematic :S

Also one module was not added to the `default` profile

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `main` branch!**

- [ ] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

</details>

### CODE REVIEW

**Review by evacchi — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -215,6 +215,7 @@
         <module>serverless-workflow-github-showcase</module>
         <module>serverless-workflow-order-processing</module>
         <module>serverless-workflow-funqy</module>
+        <module>serverless-workflow-temperature-conversion</module>
```
Comment (radtriste) [inline pom.xml]: @ricardozanini was that on purpose that this module is not included into `default` profile ?

Comment (radtriste) [reply pom.xml]: removed it. it should be solved by https://issues.redhat.com/browse/KOGITO-6060

**Comment by ricardozanini** [comment]
Temperature conversion problem solved in https://github.com/kiegroup/kogito-examples/pull/915

**Review by ricardozanini — APPROVED** [review]

**Comment by ricardozanini** [comment]
@radtriste I believe we are ready. In your morning can you please merge this and kick the CI on mine? :D

**Pull request merged**